### DISCUSSION TITLE
Recourses to learn Geospatial data science with python

### DISCUSSION THREAD

**Discussion by dahlalex** [opengeos/leafmap#216] [General]
Any recommendations for good recourses about learning geospatial data science with python ?

**Comment by giswqs** [comment]
There is an open-access book titled _Geographic Data Science with Python_
https://geographicdata.science/book/intro.html

**Comment by dahlalex** [comment]
Here are two fantastic courses organized by the Department of Geosciences and Geography at the University of Helsinki.
[Automating GIS-processes](https://autogis-site.readthedocs.io/en/latest/)
[Geo-Python](https://geo-python-site.readthedocs.io/en/latest/)
